week-3/C-array-every/exercise.js

```diff
@@ -4,8 +4,13 @@
 
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
+function checkingGroup(name) {
+  if (students.includes(name)) {
+    return name;
```

Comment:
I think this should return a boolean value.

Reply:
No, it should return either "The group contains students only or "The group does not contain students only"  have a look at the whole code, it has a console.log statement

Reply:
But you're returning the name. You should be able to do this:

```
function checkingGroup(name) {
    return students.includes(name);
}
```

Reply:
yeah you are right, I can do `return students.includes(name);` instead of the if statement